lib/engines/bh.js

```diff
@@ -7,37 +7,49 @@ module.exports = function (opts) {
 
     /**
      * bh.js express view engine adapter
-     * @api
-     * @returns {function(string, Object, Function)}
+     * Produces html from bh-templates and data and invokes passed callback with result
+     *
+     * @param   {String}     name       bundle name
+     * @param   {Object}     [options]  data to use in templates
+     * @param   {Function}   cb         callback to invoke with result or error
+     * @returns {Undefined}             undefined value
      */
     function bh(name, options, cb) {
+        var filePath;
+        var bhModule;
+        var html;
```

Comment:
Because its good practice to define all variable declaration in one place. But I don't insist on, it's only my habit.